### DISCUSSION TITLE
Turing-Nature workshops on physics and AI

### DISCUSSION THREAD

**Discussion by RaoOfPhysics** [alan-turing-institute/asg-research-communications#72] [Events]

**Comment by malvikasharan** [comment]
@ZaynabIsmail do we have more information on this yet? Can you please loop me in as needed (especially if any community/event planning related work emerge)?

**Comment by ZaynabIsmail** [comment]
@malvikasharan and @RaoOfPhysics The initial discussion with Iulia Georgescu (Nature Chief Editor) regarding the goals, format and dates for the seminars and round table has been completed and can be reviewed via the link below.  The white paper is a result of these discussions.

I have asked Hannah and Achintya to review the summary below and provide feedback if needed by Wednesday 13th July.

On a side note I would like some clarity as to the specific role Achintya will have in all these activities.  If it has already been outlined please direct me to the document.

https://thealanturininstitute.sharepoint.com/:w:/s/rid/ER2KmwDQGilDkB-OKOpwmzMBWkqMqg1FLiuoP5dEy4PSLA?e=XENdbw

**Comment by RaoOfPhysics** [comment]
Thanks for sharing the document, @ZaynabIsmail. I’ve had a look and the goals and format make sense to me. Only slightly concern I have (which has indeed been acknowledged in the document) is the US-heavy focus of the seminar speakers, which is slightly odd given that the events will be hosted by two UK-based organisations. For particle physics, at least, I know of other Europe-based researchers whom we could potentially contact.

Regarding my role, as clarified with Ben, I’ll be attending the seminars and workshops, and will be helping with drafting the document to be published in _Nat Rev Phys_ but will not be otherwise deeply involved with organising the seminars and workshops.

**Reply by ZaynabIsmail** [reply]
Please send over anyone you feel should be in attendance to the seminars  and round tables. See 'Local participants' in this [list](https://thealanturininstitute.sharepoint.com/:w:/s/rid/EXF52NIIscFLhOTL1YBsFqgBNXqM3Y2-lYbCflEydqJ8Xg?e=EcqAoi) for other people we will invite to contribute to the discussion.

- [ ] @RaoOfPhysics Please send over any people you feel should a) be a presenter b)attend as a participant.

**Comment by ZaynabIsmail** [comment]
Update: @RaoOfPhysics I have asked Iulia to establish a framework for each seminar, key outputs she would like to see.  She will outline some questions for the speakers to answer through their presentations and for the chair to ask them.  We would like your input in these questions to ensure that the discussion is conducive to generating content for the white paper.  

She has just begun to contact speakers and will create the guidance after they accept.  Once I have this I will send them to you to review.

- [ ] @ZaynabIsmail  to send draft discussion and presentation framework for Turing-Nature seminars to @RaoOfPhysics
